packages/block-directory/src/store/actions.js

```diff
@@ -94,18 +95,19 @@ export function* installBlockType( block ) {
  * @param {Object} block The blockType object.
  */
 export function* uninstallBlockType( block ) {
-	const { id } = block;
+	const plugin = block.plugin.replace( '.php', '' );
```

Comment:
This is somewhat unexpected, is the plugin file the only way to get the slug?

Reply:
Right now the plugins controller returns the file. This could be bypassed by using the `self` link. 

I don't know if the controller should return the file ie `my-plugin/init.php` or `my-plugin/init` for the plugin field. The former is the "real" value, but the latter is what will get used in the URL. I'm very much open to thoughts.